web/Controllers/LandlordController.cs

```diff
@@ -58,15 +58,25 @@ public async Task<ActionResult> RegisterPost([FromForm] LandlordProfileModel cre
         {
             case ILandlordService.LandlordRegistrationResult.Success:
                 _logger.LogInformation("Successfully created landlord for user {UserId}", user.Id);
-                string msgBody = $"A Landlord has just registered\n"
-                                 + "\n"
-                                 + $"Title: {createModel.Title}\n"
-                                 + $"First Name: {createModel.FirstName}" + "\n"
-                                 + $"Last Name: {createModel.LastName}" + "\n"
-                                 + $"Company Name: {createModel.CompanyName}" + "\n"
-                                 + $"Email: {createModel.Email}" + "\n"
-                                 + $"Phone: {createModel.Phone}" + "\n";
-                _mailService.SendMsg(msgBody);
+                var msgBody = $"A Landlord has just registered\n"
+                              + "\n"
+                              + $"Title: {createModel.Title}\n"
+                              + $"First Name: {createModel.FirstName}" + "\n"
+                              + $"Last Name: {createModel.LastName}" + "\n"
+                              + $"Company Name: {createModel.CompanyName}" + "\n"
+                              + $"Email: {createModel.Email}" + "\n"
+                              + $"Phone: {createModel.Phone}" + "\n";
+                var subject = "Bricks&Hearts - landlord registration notification";
+                try
+                {
+                    await _mailService.SendMsg(msgBody,subject);

```

Comment:
This will wait for the email to be sent (or fail to send) before continuing with this method.

If we instead do
```cs
Task.Run(_mailService.SendMsg(msgBody,subject));
```
then this method will carry on and respond to the user while the email is still being sent. Don't actually change the code here because then the exception will not be caught - instead wrap the try/catch in another function and `Task.Run` that